microsoft-edge/devtools-guide-chromium/json-viewer/json-viewer.md

```diff
@@ -88,3 +88,51 @@ To view invalid JSON data:
 1. In a code editor, such as Microsoft Visual Studio Code, go to the indicated column number to inspect the JSON:
 
    ![Viewing the malformed JSON by column number in VS Code](./json-viewer-images/column-number.png)
+
+
+<!-- ====================================================================== -->
+## Search the JSON data
+
+To search for text within the JSON data, use the **JSON viewer**'s search feature:
+
+1. View a JSON response or file in the **JSON viewer**.
+
+1. Put focus in the JSON data by clicking anywhere in the formatted JSON data.
+
+1. Press **Ctrl+F** (Windows, Linux) or **Command+F** (macOS). The search box appears at the top of the **JSON viewer**:
+  
+   ![The search feature of the JSON viewer](./json-viewer-images/search.png)
+
+1. Start typing the text you want to search for.
+
+   As you type, the **JSON viewer** highlights the occurrences of the text in the JSON data.
+
+1. To stop search, click the **Cancel** button:
+
+   ![The Cancel button in the JSON viewer's search toolbar](./json-viewer-images/search-cancel.png)
+
+
+<!-- ------------------------------ -->
+#### Move through the search results
+
+To move to the next or previous search results, after results were found, click the **Next** or **Previous** buttons near the search box, the current search result is highlighted in a different color:
+
+![The Next and Previous buttons in the JSON viewer's search toolbar](./json-viewer-images/search-move.png)
+
+
+<!-- ------------------------------ -->
+#### Change how the search is performed
```

Comment:
re-sequence the section to have a full discussion of the first checkbox & default behavior for it; and then gather statements about 2nd checkbox/default; & finally, gather statements about the 3rd checkbox.  I might push a commit to see what this specifically looks like.  avoiding an antipattern:

Point A
Point B
Point C

More about point A
More about point B
More about point C

Reply:
```
<!-- ------------------------------ -->
#### Modify the type of search

The search toolbar contains **Match Case**, **Regexp**, and **By Word** checkboxes, to modify the type of search:

![The Match Case, Regexp, and By Word buttons, which change how the search is performed](./json-viewer-images/search-modes.png)


<!-- ---------- -->
###### Perform a case-sensitive search

By default, the search is case-insensitive.  For example, a search on `hello` matches `Hello`, `HELLO`, `hello`, or any string that contains the characters `hello` with any casing.

To make the search case-sensitive, select the **Match Case** checkbox in the search toolbar.


<!-- ---------- -->
###### Search by a regular expression

By default, the search string is treated as a literal string.

To search by using a regular expression, select the **Regexp** checkbox in the search toolbar.


<!-- ---------- -->
###### Search by whole word only

By default, the search is not restricted to finding just the whole, isolated word surrounded by spaces.  For example, a search on `hello` matches `hello`, `hello-world`, and `world-hello`, or any string that contains the sequence of characters `hello`.

To restrict the search to match only the string surrounded by spaces, without adjacent letters, select the **By Word** checkbox in the search toolbar.
```